src/appleseed/renderer/modeling/object/meshobjectprimitives.cpp

```diff
@@ -0,0 +1,251 @@
+
+//
+// This source file is part of appleseed.
+// Visit http://appleseedhq.net/ for additional information and resources.
+//
+// This software is released under the MIT license.
+//
+// Copyright (c) 2016 Ramon Blanquer
```

Comment:
The copyright notice should be

    // Copyright (c) 2016 Ramon Blanquer, The appleseedhq Organization